### ISSUE TITLE
Regenerate gitwash docs for improved syntax highlighting of non-python code sections

### ISSUE THREAD

**Issue by anntzer** [matplotlib/matplotlib#8123]
See https://github.com/matthew-brett/gitwash/issues/10.

**Comment by aashil** [comment]
Hi @anntzer, I would like to work on this one.

**Comment by anntzer** [comment]
Go for it :)

**Comment by NelleV** [comment]
While you are at it, do you mind adding a readme in doc/devel indicating that the gitwash folder is automatically generated and should not be modified. It is also a good place to write down the command line you are using to generate the folder.

**Comment by aashil** [comment]
Sure.

**Comment by aashil** [comment]
Just want to make sure if I am using the correct command. Can you please confirm ? 
```
python gitwash_dumper.py .. Matplotlib --repo-name=matplotlib --github-user=matplotlib \
   --project-url=http://matplotlib.org \
   --project-ml-url=https://mail.python.org/mailman/listinfo/matplotlib-devel
```

**Comment by anntzer** [comment]
You probably want to look up who did this first for this repo, and ping him to confirm (I have no idea, I guess it looks reasonable).

**Comment by NelleV** [comment]
This looks reasonable :)

**Comment by NelleV** [comment]
actually, there is a problem
the output directory should be gitwatsh not ..

**Comment by aashil** [comment]
Yes, I changed it. I downloaded `gitwash_dumper.py` in `doc/devel/gitwash` hence `..`.

**Comment by aashil** [comment]
FYI, gitwash was first implemented in [fa26fa11](https://github.com/matplotlib/matplotlib/commit/fa26fa116c9ca18b327f219bc9d8e30bc298ca35). Sent a quick email to the contributor. Let's see if he can help else will go with the above script.
